ops-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM maven:3.8.4-jdk-11-slim AS build
+COPY target/docker-practice-java-0.0.1-SNAPSHOT.jar /home/app/app.jar
```

Comment:
This line assumes that we have jar file locally. But it means that we have to install java locally to build this jar file. It should be enough to have only docker and docker-compose for running application locally. So now if I clone you repo and try to run docker-compose up I will have a error: file does not exist.

Reply:
Fixed remarks

Reply:
Cool, thank you! Your work is accepted